### PR TITLE
[FLINK-22879][table] Remove "blink" suffix from table modules

### PR DESCRIPTION
**Pull request by twalthr**
## What is the purpose of the change

This removes the `blink` suffix from all table modules and artifacts. The API itself is not affected by this change, however, it will cause to adjust dependencies for some users.

This PR is based on #16374.

## Brief change log

See commit messages.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): yes
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: yes
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by twalthr** [comment]
@wuchong this causes a lot of merge conflicts, a quick review would be appreciated

File: docs/content/docs/dev/table/sql/set.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ The following examples show how to run a `SET` statement in SQL CLI.
 {{< tabs "set" >}}
 {{< tab "SQL CLI" >}}
 ```sql
-Flink SQL> SET 'table.planner' = 'blink';
+Flink SQL> SET 'table.local-time-zone' = 'Europe/Berlin';
 [INFO] Session property has been set.
 
 Flink SQL> SET;
-'table.planner' = 'blink'
+'table.planner' = 'Europe/Berlin'
```
Comment (wuchong) [inline docs/content/docs/dev/table/sql/set.md:50]: `'table.local-time-zone' = 'Europe/Berlin'`

**Review by wuchong — COMMENTED** [review]
Looks good to me. I only left one minor comment.

**Comment by twalthr** [comment]
Merging...

**Pull request closed without merging**